editor.planx.uk/src/@planx/components/Send/uniform/UniformPayload/schema.ts

```diff
@@ -0,0 +1,247 @@
+import { z } from "zod";
+
+// Models the XML payload required by Uniform to process PlanX applications
+
+export const paymentSchema = z.object({
```

Comment:
The contents of this file were generated by just pasting the existing types into https://transform.tools/typescript-to-zod

I think there's a viable (if slightly annoying) method where we could enrich this by going XSD -> JSON Schema -> Zod